color/src/colorspace.rs

```diff
@@ -525,6 +525,61 @@ impl ColorSpace for Rec2020 {
     }
 }
 
+/// 🌌 The ACEScg color space.
+///
+/// This color space is defined by the Academy Color Encoding System [specification][acescg].
+///
+/// The ACEScg color space is a linear color space. The wide gamut makes this color space useful as
+/// a working space for computer graphics.
+///
+/// Other ACES color spaces include ACES2065-1, ACEScc, and ACEScct. This crate currently does not
+/// support these color spaces.
+///
+/// The ACEScg components are `[R, G, B]`. The components are bounded to `[-65504.0, 65504.0]`,
+/// though it is unusual to clip in this color space.
+///
+/// The conversion between D65 and the ACES white point is done using the standard Bradford
+/// linear chromatic adaptation transform. See the [XYZ-D65 color space](`XyzD65`) documentation for
```

Comment:
The choice so far has been to not go into too much detail about white point in the individual color spaces, Raph expressed that preference. We've used XYZ-D65 as a place to expand on the topic a bit. I think you can copy the wording from for example ProPhoto RGB about the white point.

```
/// ProPhoto RGB has a reference white of
/// D50; see the [XYZ-D65 color space](`XyzD65`) documentation for some background information on
/// the meaning of "reference white."
```